reference/tools/build.rst

```diff
@@ -138,6 +138,12 @@ conan.tools.build.valid_max_cstd()
 
 .. autofunction:: valid_max_cstd
 
+conan.tools.build.default_cstd()
+""""""""""""""""""""""""""""""""""
+
+.. currentmodule:: conan.tools.build.cstd
```

Comment:
The ``conan.tools.build.cstd`` shouldn't probably be used externally? Isn't it possible to declare here ``conan.tools.build`` similar to the user imports?